assets/js/components/ExecutionResults/ExecutionResultsPage.jsx

```diff
@@ -38,17 +37,13 @@ function ExecutionResultsPage() {
   if (!cluster) {
     return <div>Loading...</div>;
   }
+  const clusterSelectedChecks = cluster?.selected_checks;
```

Comment:
I don't think we need a new constant here. Simply pass `cluster?.selected_checks` to `clusterSelectedChecks` as we do with other props

Reply:
Oki yeah!

Reply:
Yeah sure :D